pom.xml

```diff
@@ -55,18 +55,18 @@
       <url>https://oss.sonatype.org/content/repositories/snapshots</url>
     </snapshotRepository>
     <site>
-      <id>cask</id>
-      <url>http://cask.co</url>
+      <id>cdap</id>
+      <url>http://cdap.io</url>
     </site>
   </distributionManagement>
 
   <issueManagement>
-    <url>https://issues.cask.co/browse/HYDRATOR</url>
+    <url>https://issues.cask.co/browse/CDAP</url>
   </issueManagement>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
```

Comment:
hmmm, in release branch it shouldn't be snapshot, right?

Reply:
That is intentional otherwise compilation would fail since we have not released 6.0.0 artifacts of cdap yet.

Reply:
is that what we're doing everywhere? If so, merge.

Reply:
that is same everywhere. Also without snapshot version, compilation will fail.